spec/controllers/api/v1/changesets_controller_spec.rb

```diff
@@ -30,7 +30,7 @@
       create_list(:changeset_with_payload, 2)
       changeset = Changeset.last
       get :show, id: changeset.id
-      expect_json({
+      expect_json('changeset', {
```

Comment:
Redundant curly braces around a hash parameter.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.